src/hr/hr.controller.ts

```diff
@@ -17,4 +26,18 @@ export class HrController {
   create(@Body() newHr: AddHrDto) {
     return this.hrService.addHr(newHr);
   }
+
+  @Roles(Role.HR)
+  @UseGuards(AuthGuard('jwt'), RoleGuard)
+  @Patch()
+  addStudent(@UserObj() hr: User, @Body() student: { email: string }) {
+    return this.hrService.addStudent(student.email, hr);
+  }
+
+  @Roles(Role.HR)
+  @UseGuards(AuthGuard('jwt'), RoleGuard)
+  @Delete()
+  removeStudent(@UserObj() hr: User, @Body() student: { email: string }) {
+    return this.hrService.removeStudent(student.email, hr);
+  }
```

Comment:
Missed endpoint for user(student) with status hired

Reply:
Added endpoint for hiring students with email information and own email template. Changed the http method in removeStudent from DELETE to PATCH. Added validation in removeStudent when hr tries to remove already hired student from the list